python/golem_task_api/apputils/logging.py

```diff
@@ -0,0 +1,52 @@
+import logging.config
```

Comment:
IMHO this file should be named `logging_.py` to avoid problems with tools that are not yet fully python3 compatible (mypy etc.)

Reply:
hmm, would `logger.py` also work? like that more then adding an _

Reply:
logger is also already somewhat a reserved name, settled on `start_logging.py`